lib/rubocop/rspec_rails/shared_contexts/default_rspec_language_config_context.rb

```diff
@@ -1,15 +1,15 @@
 # frozen_string_literal: true
 
-RSpec.shared_context 'with default RSpec/Language config' do
+RSpec.shared_context 'with default RSpecRails/Language config' do
   include_context 'config'
 
   # Deep duplication is needed to prevent config leakage between examples
   let(:other_cops) do
     default_language = RuboCop::ConfigLoader
-      .default_configuration['RSpec']['Language']
+      .default_configuration['RSpecRails']['Language']
```

Comment:
Where would this come from? We shouldn’t have any RSpecRails entry in the config

Reply:
Oops, I removed it. Thank you.